test_sorting_insertionsort.py

```diff
@@ -15,3 +15,44 @@ def test_randoms():
 def test_string():
     with pytest.raises(TypeError):
         insertion_sort('cat')
+
+
+def test_empty():
+    x = []
+    y = x
+    insertion_sort(x)
+    assert x == y
+
+
+def test_reversed():
+    x = []
+    for i in range(1, 10):
+        x.append(i)
+    y = []
+    for i in reversed(x):
+        y.append(i)
+    insertion_sort(y)
+    z = y
+    assert z == x
```

Comment:
This could be a 4 line test. The logic is too verbose and can be so much cleaner.